### ISSUE TITLE
VS fails to attach to service running under a LocalUser account

### ISSUE THREAD

**Issue by johnstairs** [Azure/service-fabric-issues#455]
Consider the following snippet from an ApplicationManifest file. Note that there are two users, one of which is a LocalUser (the default), and the other Network Service. Also note that the user that the service run as is controlled by the [FrontendService_UserAccount] parameter value.

```  
...
<ServiceManifestImport>
  <ServiceManifestRef ServiceManifestName="FrontendServicePkg" ServiceManifestVersion="1.0.0" />
  <ConfigOverrides />
  <Policies>
    <RunAsPolicy CodePackageRef="Code" UserRef="[FrontendService_UserAccount]" />
    <EndpointBindingPolicy EndpointRef="ServiceEndpoint" CertificateRef="SslCert" />
    <SecurityAccessPolicy ResourceRef="ServiceEndpoint" PrincipalRef="[FrontendService_UserAccount]" />
  </Policies>
</ServiceManifestImport>
<DefaultServices>
  <Service Name="FrontendService" ServicePackageActivationMode="ExclusiveProcess">
    <StatelessService ServiceTypeName="FrontendServiceType" InstanceCount="[FrontendService_InstanceCount]">
      <SingletonPartition />
      <PlacementConstraints>[FrontendService_PlacementConstaints]</PlacementConstraints>
    </StatelessService>
  </Service>
</DefaultServices>
<Principals>
  <Users>
    <User Name="CloudAccount" />
    <User Name="LocalAccount" AccountType="NetworkService" />
  </Users>
</Principals>
...
```
If the parameter is CloudAccount, meaning the service runs under a LocalUser account, and I hit F5 in Visual Studio, VS does not attach to the service process (running under this user account). I need to explicitly attach to the process. 

If, however, the parameter is LocalAccount, meaning the process runs as Network Service, everything works fine.

In the above, you can see my workaround: parameterize the account to use, so we use network service locally, and a local user in the cloud :)

I'm using VS 2017.

**Comment by dbreshears** [comment]
Sorry for the delay inn the response.   This is a bug in the VS debugger that will be fixed in a future VS update.  Currently set for 15.6

**Comment by dbreshears** [comment]
@johnstairs , this appears to have been fixed in 15.6.  Closing, please re-open if you find otherwise.
